IRremoteESP8266.cpp

```diff
@@ -749,10 +749,7 @@ static void ICACHE_RAM_ATTR gpio_intr() {
     irparams.rcvstate = STATE_MARK;
     irparams.rawbuf[irparams.rawlen++] = 1;
   } else {
-    if (now < start)
-      irparams.rawbuf[irparams.rawlen++] = (0xFFFFFFFF - start + now) / USECPERTICK + 1;
-    else
-      irparams.rawbuf[irparams.rawlen++] = (now - start) / USECPERTICK + 1;
+    irparams.rawbuf[irparams.rawlen++] = now - start;
```

Comment:
micros() returns a uint32_t (unsigned long), currently rawbuf[] is a uint16_t (unsigned int).
i.e. 2 -> 4 bytes per entry if we move to just storing micros, not the deltas.
My guess is an assumption that was made along the way somewhere was that the differences(deltas) would always be fit in a 16bit value. It's a fairly safe assumption, because at 0xFFFF (65535) usecs, it would be a 65ms gap in an IR command, and fairly safe to assume the command had ended at that stage.